avni-server-api/src/main/java/org/avni/web/api/ProgramEnrolmentFieldNames.java

```diff
@@ -10,4 +10,6 @@ public class ProgramEnrolmentFieldNames {
     public static final String EXIT_DATETIME = "Exit datetime";
     public static final String EXIT_LOCATION = "Exit location";
     public static final String EXIT_OBS = "exitObservations";
+    public static final String EXTERNAL_ID = "External Id";
+    public static final String SUBJECT_EXTERNAL_ID = "Subject external Id";
```

Comment:
since we are using sentence case we can rename to "External id" and "Subject external id"

Reply:
Actually, we should change it to "External ID" and "Subject external ID" keeping in sync with the way rest of the ID fields are named. 
Ex:  @JsonProperty("Subject ID")
    private String subjectId;